x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -205,10 +199,6 @@ public class TokenService {
     static final int MINIMUM_BYTES = VERSION_BYTES + TOKEN_LENGTH + 1;
     static final int LEGACY_MINIMUM_BASE64_BYTES = Double.valueOf(Math.ceil((4 * LEGACY_MINIMUM_BYTES) / 3)).intValue();
```

Comment:
We can remove `LEGACY_MINIMUM_BASE64_BYTES` and `LEGACY_MINIMUM_BYTES` as well since they are not referenced anymore.